test/envoye2e/stackdriver_plugin/stackdriver_test.go

```diff
@@ -26,6 +26,8 @@ import (
 
 func enableStackDriver(t *testing.T, vars map[string]string) {
 	t.Helper()
+	// Disable flakey CI Tsan/Asan tests
```

Comment:
Is the reason to skip the tsan/asan tests here: the flaky tests cause the automator tests to fail and need to manually rerun the tests, which is time consuming and requires manual operations? 
Can you add the reasons to the code comment?

Reply:
From the test failure below, test-asan_proxy may still occur.

The following test failed, say /retest to rerun all failed tests or /retest-required to rerun all mandatory failed tests:

Test name	Commit	Details	Required	Rerun command
test-asan_proxy	https://github.com/istio/proxy/commit/c8c15932796ee90ecff9d23c9696c17a5c587300	[link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_proxy/5145/test-asan_proxy/1724583108281372672)	true	/test test-asan